datastream-server/src/main/java/com/linkedin/datastream/server/assignment/StickyPartitionAssignmentStrategy.java

```diff
@@ -29,16 +29,34 @@
 import com.linkedin.datastream.server.DatastreamTask;
 import com.linkedin.datastream.server.DatastreamTaskImpl;
 
+import static com.linkedin.datastream.server.assignment.BroadcastStrategyFactory.CFG_MAX_TASKS;
+import static com.linkedin.datastream.server.assignment.StickyPartitionAssignmentStrategyFactory.CFG_PARTITIONS_PER_TASK;
+import static com.linkedin.datastream.server.assignment.StickyPartitionAssignmentStrategyFactory.CFG_PARTITION_FULLNESS_THRESHOLD_PCT;
+
+
 /**
  *
- * The StickyPartitionAssignmentStrategy extends the StickyMulticastStrategy but allows to perform the partition
+ * The StickyPartitionAssignmentStrategy extends the StickyMulticastStrategy to allow performing the partition
  * assignment. This StickyPartitionAssignmentStrategy creates new tasks and remove old tasks to accommodate the
  * change in partition assignment. The strategy is also "Sticky", i.e., it minimizes the potential task mutations.
  * The total number of tasks is also unchanged during this process.
+ *
+ * If elastic task assignment is enabled then the initial number of tasks to create, when no tasks exist for a given
+ * DatastreamGroup, is determined by a "minTasks" datastream metadata property. When the first partition assignment
+ * takes place, based on the "partitionsPerTask" and "partitionFullnessFactorPct", the number of tasks needed to host
+ * the partitions is determined. An in-memory map tracking the number of tasks per DatastreamGroup is updated and a
+ * DatastreamRuntimeException is thrown to retry task assignment with the updated number of tasks. The next partition
+ * assignment should determine that the number of tasks are sufficient, and should go through successfully. After the
```

Comment:
nit: comma before and.

Reply:
did you mean remove the comma? removed it